src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -525,12 +525,19 @@ def _deploy_workflow(self, step_name: str, settings):
         return None
 
     def _upload_wheel(self):
+        wheel_paths = []
         with self._wheels:
-            return self._wheels.upload_to_wsfs()
-
-    def _upload_wheel_runner(self, remote_wheel: str):
+            if self._config.upload_dependencies:
+                wheel_paths = self._wheels.upload_wheel_dependencies(
+                    ["databricks_sdk", "sqlglot"]
+                )  # TODO: add more dependencies if required
+            wheel_paths.append(self._wheels.upload_to_wsfs())
+            return wheel_paths
```

Comment:
prepend `/Workspace` prefix in this method instead of doing it in `_upload_wheel_runner` and `_job_wheel_task`